tests/unit/amo/components/TestGetFirefoxBanner.js

```diff
@@ -128,7 +133,7 @@ describe(__filename, () => {
     });
 
     it('sets the href on the button with the expected utm params when there is no variant', () => {
-      const root = render({ store });
+      const root = render({ store, variant: undefined });
```

Comment:
Why is this needed now?

Reply:
With the experiment enabled globally (which includes in the tests), the `withExperiment` HOC is now running and calculating a `variant` just as it is meant to. This means that if we do not manually override `variant` with `undefined` in `render`, then the HOC assigns a variant and this test is no longer testing the case of "when there is no variant". This was passing before, without the explicit override of `variant`, because the experiment was not enabled, and therefore the HOC returned `undefined` for `variant`.